lib/champaign_queue/clients/sqs.rb

```diff
@@ -0,0 +1,37 @@
+require 'aws-sdk'
```

Comment:
I'm not sure this directory structure makes sense to me intuitively. Will ChampaignQueue have more clients than just the SQS client? If we'll want to store AWS client classes logically then why not have them in lib/aws/clients?